fluentui_persona/src/main/java/com/microsoft/fluentui/persona/AvatarView.kt

```diff
@@ -148,6 +147,20 @@ open class AvatarView : AppCompatImageView {
             invalidate()
         }
 
+    /**
+     * Defines the [ContentDescription] applied to the avatar.
+     */
+    var avatarContentDescriptionLabel: String = ""
```

Comment:
will this still read the control type out as button to signify that it is actionable? I presume it will...

Reply:
Yea, it will announce it as button. If we have a clicklistener assigned to it. if not, it will just announce the label

Reply:
great, thanks for confirming!